src/backend/management/commands/run_telegram_bot.py

```diff
@@ -113,18 +116,19 @@ async def start_individual_order(update: Update, context: ContextTypes.DEFAULT_T
     pedigroup_group = Group.objects.get(id_app=group_id)
     group_name = pedigroup_group.name
     reply_markup = InlineKeyboardMarkup(
-        [[InlineKeyboardButton("Realizar pedido individual", callback_data=f"pedir {group_id} {group_name}")]])
+        [[InlineKeyboardButton("Realizar pedido individual", callback_data=f"pedir YES {group_id} {group_name}")]])
     await update.message.reply_text(f"Bienvenido a PediGroup, queres añadir pedidos individuales para _{group_name}_?",
                                     reply_markup=reply_markup, parse_mode="Markdown")
 
 
 async def show_initial_restaurants(update: Update, context: ContextTypes.DEFAULT_TYPE):
     query = update.callback_query
     await query.edit_message_reply_markup(reply_markup=None)
-    message_with_group_id_and_name = query.data.removeprefix("pedir ")
-    message_with_group_id_and_name = message_with_group_id_and_name.split(" ")
-    group_id = int(message_with_group_id_and_name[0])
-    group_name = ' '.join(message_with_group_id_and_name[1:])
+    message_with_flag_group_id_and_name = query.data.removeprefix("pedir ")
+    message_with_flag_group_id_and_name = message_with_flag_group_id_and_name.split(" ")
```

Comment:
En vez de pisar la variable siempre no se podría hacer el split directo?
```suggestion
    message_with_flag_group_id_and_name = query.data.removeprefix("pedir ").split(" ")
```